demo/app.js

```diff
@@ -42,13 +93,23 @@ class App extends React.Component {
     this.transcriptEditorRef = React.createRef();
   }
 
-  loadDemo = () => {
+  loadDemo = (demoId) => {
+    const demo = DEMOS.find((d)=>{
+      return d.id === demoId;
+    })
+    console.log(demo)
+    const DEMO_TRANSCRIPT = demo.json;
+    const DEMO_MEDIA_URL = demo.url;
+    const DEMO_TITLE = demo.title;
+    const DEMO_TYPE = demo.type
+
       if(this.state.useLocalStorage && isPresentInLocalStorage(DEMO_MEDIA_URL)){
```

Comment:
Yeah the flow would be, untick `user local storage` then click `load demo`.
Coz I have had people using the transcript editor page, in conjunction with autoEdit to correct transcripts, and didn't want to disable the local storage default auto save, just in case they were still using it. 

But yeah, it's not ideal UX, probl untick as default would be better for development?

Reply:
yeah seems sensible to untick by default